docs/multicluster/user-guide.md

```diff
@@ -592,43 +633,149 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
-## Multi-cluster Pod-to-Pod Connectivity
+## Multi-cluster Stretched NetworkPolicy
 
-Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters
-through Multi-cluster Gateways. Pod IPs can be reached in all member clusters
-within a ClusterSet. To enable this feature, the cluster's Pod CIDRs must be set in
-ConfigMap `antrea-mc-controller-config` of each member cluster like the example
-below. Note, **Pod CIDRs must not overlap among clusters to enable cross-cluster
-Pod-to-Pod connectivity**.
+Antrea-native policies can be enforced on cross-cluster traffic in a ClusterSet.
+To use this feature set, make sure to check the Antrea controller and agent ConfigMaps
+and confirm that `enableStretchedNetworkPolicy` is set to `true` in addition to enabling
+multicluster feature:
+
+```yaml
+antrea-agent.conf: |
+  # FeatureGates is a map of feature names to bools that enable or disable experimental features.
+  featureGates:
+...
+    Multicluster: true
+...
+  multicluster:
+    enable: true
+    namespace: ""
+    enableStretchedNetworkPolicy: true
+```
+
+```yaml
+antrea-controller.conf: |
+  # FeatureGates is a map of feature names to bools that enable or disable experimental features.
+  featureGates:
+...
+    Multicluster: true
+...
+  multicluster:
+    enableStretchedNetworkPolicy: true
+```
+
+The `enableStretchedNetworkPolicy` option also needs to be set to `true` in `antrea-mc-controller-config`
```

Comment:
This should be moved to the "Stretched NetworkPolicy" section, as it is not needed by egress policies to "Multi-cluster Services"?

Reply:
Done